docs/content/dagster-cloud/deployment/serverless.mdx

```diff
@@ -89,9 +89,36 @@ dagster-cloud serverless deploy \
 
 Often you'll need to securely access secrets from your jobs. With Dagster Cloud Serverless, secrets are added as environment variables that can be read using APIs like [StringSource](/guides/dagster/transitioning-data-pipelines-from-development-to-production#part-two-deployment).
 
-- If you're using our GitHub integration, you can pass in secrets as a JSON string input to the serverless deploy step in your repository's Github action script.
+- If you're using our GitHub integration, you can create secrets in GitHub and then pass those secrets to Dagster by modifying the serverless deploy step in your repository's GitHub action script.
+
+```shell
+# .github/workflows/deploy.yml AND .github/workflows/branch_deployments.yml
+...
+jobs:
+  ...
+  dagster_cloud_build_push:
+  ...
+    steps:
+    ...
+    - name: Build and deploy to Dagster Cloud serverless
+          uses: dagster-io/dagster-cloud-action/actions/serverless_prod_deploy
+          with:
+            dagster_cloud_api_token: ${{ secrets.DAGSTER_CLOUD_API_TOKEN }}
+            location: ${{ toJson(matrix.location) }}
+            # Uncomment to pass through Github Action secrets as a JSON string of key-value pairs
+            # env_vars: ${{ toJson(secrets) }}
+```
+
 - If you're using the CLI directly, you can pass `KEY=VALUE` pairs using the `--env` flag. Run `dagster-cloud serverless deploy --help` to learn more.
 
+### Adding Non-Python Dependencies
+
+While Python dependencies will be installed via `setup.py` or `requirements.txt`, occasionally a project will require additional dependencies. For example, dbt dependencies that are installed with `dbt deps`.
```

Comment:
Believe we have taken out requirements.txt in the quickstart repos since having both was confusing, i think you can say setup.py here only?